src/pages/Congratulations/index.tsx

```diff
@@ -147,21 +149,31 @@ const _CongratulationsPage = ({
                 />
                 <div className="flex space-between mt20">
                   <ProgressBarInfo
-                    title="Staking balance:"
+                    title={formatMessage({
+                      defaultMessage: 'Staking balance:',
+                      description: 'Informing user of staking balance',
+                    })}
                     color={colors.blue.dark}
                     amountEth={amountEth}
                     // @ts-ignore
                     amountValidators={amountEth / PRICE_PER_VALIDATOR}
                   />
                   <ProgressBarInfo
-                    title="You added:"
+                    title={formatMessage({
+                      defaultMessage: 'You added:',
+                      description: 'Informing user of total funds deposited',
+                    })}
                     color={colors.blue.light}
                     // @ts-ignore
                     amountEth={depositKeys.length * PRICE_PER_VALIDATOR}
                     amountValidators={depositKeys.length}
                   />
                   <ProgressBarInfo
-                    title="Launch threshold:"
+                    title={formatMessage({
+                      defaultMessage: 'Launch threshold:',
+                      description:
+                        'Refers to threshold of ETH needed to launch the beacon chain',
+                    })} /* TODO: remove this reference? */
```

Comment:
@samajammin @ryancreatescopy Another instance of this that should be removed if you agree.